tests/test_fingerprints.py

```diff
@@ -0,0 +1,41 @@
+"""Test for fingerprint generation."""
+
+from baybe.parameters import SubstanceEncoding
+from baybe.utils.chemistry import smiles_to_fingerprint_features
+
+
+def test_fingerprint_computation():
```

Comment:
test looks good
but please parameterize it, like eg done [here](https://github.com/emdgroup/baybe/blob/main/tests/test_objective.py#L83)

in this way you cang enerate 1 test for morgan or others where lenght of fingeprrint might not be adjustable and 2 or more tests for ECFP where length is adjustable. The shape test e.g. could be doe only when `fp_size` was in the kwargs 

```python
@pytest.mark.parametrize(
    ("name", "kwargs_fingerprint", "kwargs_conformer"),
    [
        ('MORDRED', {}, {}),
        ('ECFP', {}, {}),
        ('ECFP', {'fp_size':128}, {}),
        ...
    ],
)
def test_stuff(name, kwargs_fingerprint, kwargs_conformer)
```